flask_app/filters.py

```diff
@@ -33,6 +33,11 @@ def get_unsuccessful_filters(cls, model):
     def build_successful_query(cls, model):
         conds = cls.get_unsuccessful_filters(model)
         conds.append(model.start_time == None)
+        if model is Session:
+            conds.append(model.num_skipped_tests != 0)
```

Comment:
An "successful" session is not one that has no skips... It is one that has no errors...